src/signalSTM.h

```diff
@@ -0,0 +1,354 @@
+#pragma once
+
+#if !defined(__AVR__) && defined(ARDUINO_MAPLEMINI_F103CB) || defined(STM32F103xB)
+#include "compile_config.h"
+
+/*
+
+*   developer:
+*   2020 S.Butzek, HomeAutoUser, elektron-bbs
+
+*   Integration for compatibility with a similar project by Ralf9
+
+*   tested hardware:
+*    - Maple Mini STM32F103CBT6
+
+*/
+
+#define PROGNAME               " SIGNALduino_STM32 "
+#define PROGVERS               "3.4.0-dev_20200626"
+#define VERSION_1               0x33
+#define VERSION_2               0x1d
+
+#define BAUDRATE               115200
+#define FIFO_LENGTH            170
+#define defSelRadio 1                           // variant -> Circuit board for 4 cc110x - standard value 1 = B
+const uint8_t pinReceive[] = {11, 18, 16, 14};  // variant -> Circuit board for 4 cc110x
```

Comment:
Der RecivePin ist schon in compile_config.h definiert. Da sollten wir es auch belassen und hier keine Boardspezifischen Dinge machen. Hier sollten wir uns auf STM Generische Dinge beschänken, sonst verzetteln wir  uns nur.

Reply:
In der `compile_config.h` wurde der Pin FEST verankert für die Variante der Kompatiblität zu der Firmware von Ralf.
Wenn man die Platine mit 4 Empfängern einbinden möchte, so benötigt man alle 4.

Diese Zeile könnte man ggf auskommentieren oder man switcht diese in die `compile_config.h` um mit mehreren Varianten.
1) Variante nur 1 Empfänger = "kompatiblitätsmodus"
2) Variante mit 4 Empfänger...